### DISCUSSION TITLE
List<> 사용 방법 질문 드립니다.

### DISCUSSION THREAD

**Discussion by MAX-Jeon** [naverz/zepeto-studio-kor#956] [5. Q&A]
![image](https://user-images.githubusercontent.com/102704503/177149863-c8d2c40f-35f0-4422-90bc-7f40fe2c8aee.png)

List로 등록 후 오브젝트를 SetActive(false)된 순서대로 생성, 제거 하려고 하는데
List 함수 사용 하는 방법이나 예시가 있으면 알려주세요~!

**Comment by YSHFM** [comment]
먼저, Array<>를 쓰시거나 List$1<>을 쓰셔야 합니다.
Array나, List던, 자료형 선택 후 f12누르거나, index.d.ts 파일 내에 함수를 확인할 수 있습니다. 
push,pop, 인덱스 접근 등 다양하니 여러 방법 써보시면 될 것 같습니다.
